src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -177,17 +176,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     $row[] = $this->sanitizeString($fallbackCatalogue->get($messageId, $domain));
                 }
 
-                $table->addRow($row);
+                $rows[] = $row;
             }
         }
 
-        $table->render();
-
-        $output->writeln('');
-        $output->writeln('<info>Legend:</info>');
-        $output->writeln(sprintf(' %s Missing message', $this->formatState(self::MESSAGE_MISSING)));
-        $output->writeln(sprintf(' %s Unused message', $this->formatState(self::MESSAGE_UNUSED)));
-        $output->writeln(sprintf(' %s Same as the fallback message', $this->formatState(self::MESSAGE_EQUALS_FALLBACK)));
+        $output->section('Debug Translation result:');
```

Comment:
We should probably remove the trailing `:` for the section titles, because they are already highlighted with the underlined dashes.